examples/cfd/external_aerodynamics/domino/src/conf/config.yaml

```diff
@@ -38,9 +38,9 @@ resume_dir: ${output}/models
 # └───────────────────────────────────────────┘  
 data_processor: # Data processor configurable parameters
   kind: drivaer_aws # must be either drivesim or drivaer_aws
-  output_dir: /lustre/rranade/aws_data_all/
-  input_dir: /lustre/datasets/drivaer_aws/drivaer_data_full/
-  cached_dir: /lustre/cached/drivaer_aws/drivaer_data_full/
+  output_dir: /user/aws_data_all/
+  input_dir: /data/drivaer_aws/drivaer_data_full/
+  cached_dir: /user/cached/drivaer_aws/drivaer_data_full/
```

Comment:
We already declare both `project_dir` and `output` in this config.  Is it possible to reuse those with `${output}`?  I don't want to leave a config that requires many manual changes, it'd be nice to format all these paths as sensible relative paths.